setup.py

```diff
@@ -31,7 +31,7 @@ def find_stubs(package):
 setup(
     name='pynamodb',
     version=__import__('pynamodb').__version__,
-    packages=find_packages(),
+    packages=find_packages(exclude=('pynamodb.tests',)),
```

Comment:
```suggestion
    packages=find_packages(exclude=('pynamodb.tests', 'pynamodb.tests.*')),
```

https://setuptools.readthedocs.io/en/latest/setuptools.html#using-find-packages suggests that we'll need this to also omit the integration test package

Reply:
From manually testing this, excluding the parent package was enough to exclude all tests in the resulting dist.

Reply:
@asottile says that this will exclude from sdist too. He also thinks we should `mv pynamodb/tests tests` and then `exclude=('tests*',)`.

And, regardless, we should dig into #593 some more, since it's a hack w.r.t #593.

Reply:
I don't feel strongly about exactly where tests live, but if you've tested this and it works for subpackages too then happy to 🚢